### PR TITLE
Cleanup, polish install instructions, add to release notes

### PR DESCRIPTION
**Pull request by s7eph4n**
* remove temporary package installations from travis build script
* do not load filtered by default (it loads map data from external page)
* polish INSTALL.MD
* add changes to RELEASE-NOTES.md

This PR is made in reference to: #190 

This PR includes:
- [x] Tests (unit/integration)
- [x] CI build passed

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -139,38 +139,45 @@ minimum requirements are indicated in bold.
...
+   * Remark 1: Remember to add a comma to the end of the preceding line in this 
+     section.
 
-You should already have a "composer.local.json" file due to installing the required extension
-Semantic MediaWiki. If not you should install Semantic MediaWiki first before you continue.
+   * Remark 2: If you do not have a `composer.local.json` file (MediaWiki <1.25),
+     use `composer.json` instead.
 
-When this is done run in your shell:
+   * Remark 3: If you do not have a `composer.json` file (MediaWiki <1.23.5),
```
Comment (kghbln) [inline INSTALL.md]: This may go since this release is not supposed to support MW <1.23 any longer

**Comment by kghbln** [comment]
@s7eph4n Thanks for fluff. :)

File: INSTALL.md (hunk 1 above)
Comment (s7eph4n) [inline INSTALL.md]: But 1.23 .0 < 1.23.5. ;-)

Comment (kghbln) [inline INSTALL.md]: What a mess MW created. I figured that 1.23.x already used `composer.json` itself. This acutually started with 1.24.0. So shouldn't it be 1.23.17 ;)

Comment (s7eph4n) [inline INSTALL.md]: Not sure. I just copied over the remarks from Chameleon without checking them again. I think what happened is that at the time of writing 1.23.5 was the latest on the 1.23 branch. We should probably just put <1.24.

Comment (kghbln) [inline INSTALL.md]: Probably maybe. Or 1.23.x to make sure what people do not think of 1.22. But this is nitpicking. :)

Comment (s7eph4n) [inline INSTALL.md]: Done. In any case, it should only be in for a few months.

**Pull request merged**